tasks/test_module/my_task.py

```diff
@@ -0,0 +1,417 @@
+import csv
+import os
+
+from collections import OrderedDict
+from luigi import Task, LocalTarget
+from tasks.meta import (OBSColumn, OBSColumnToColumn, OBSTag, current_session,
+                        DENOMINATOR, GEOM_REF)
+from tasks.util import (LoadPostgresFromURL, classpath, shell, LOGGER,
+                        CartoDBTarget, get_logger, underscore_slugify, TableTask,
+                        ColumnTarget, ColumnsTask, TagsTask, TempTableTask,
+                        classpath, PostgresTarget, MetaWrapper)
+from tasks.tags import SectionTags, SubsectionTags, UnitTags, BoundaryTags
+from time import time
+
+
+class DownloadGeometry(Task):
+
+    URL = 'http://www.ine.es/censos2011_datos/cartografia_censo2011_nacional.zip'
+
+    def run(self):
+        self.output().makedirs()
+        shell('wget {url} -O {output}'.format(url=self.URL,
+                                              output=self.output().path))
+    def output(self):
+        return LocalTarget(os.path.join('tmp', classpath(self),
+                                        'cartografia_censo2011_nacional.zip'))
+
+class RawGeometry(TempTableTask):
```

Comment:
Should subclass [Shp2TempTableTask](https://github.com/CartoDB/bigmetadata/blob/master/tasks/util.py#L1336).